src/az390.java

```diff
@@ -2124,11 +2154,11 @@ else if (tz390.op_trace_type[index]==343)
                 case 36:
                     // Alternate formats have mnemonics ending in 'A'
                     if (my_mnemonic.substring(my_mnemonic.length()-1).equals("A"))     // #500
-                       {my_format="RRF";                                               // #500
+                       {my_format="RRF";                                               // #500 #573
```

Comment:
Looks the same as the previous line??? Does not appear to be changing anything, but okay.